i18n/ind/docusaurus-plugin-content-docs/current/lisk-tools/node-providers.md

```diff
@@ -0,0 +1,95 @@
+---
```

Comment:
Please remove this file, as it has already been translated and will be updated through the translation tracker.